src/indicator/renderer/relativeStrengthIndex.js

```diff
@@ -19,9 +19,17 @@ export default function() {
 
     var rsi = function(selection) {
 
-        multiSeries.xScale(xScale)
+        multi
+            .xScale(xScale)
             .yScale(yScale)
-            .series([rsiLine, annotations])
+            .series([annotations, rsiLine])
+            .decorate(function(g, data, index) {
```

Comment:
I don't think my question was clear (and probably on the wrong line!) - your commit changes the order in which the methods on the D3 selection are called. Was this required in order to fix some issue?